spec/models/user_spec.rb

```diff
@@ -917,6 +917,37 @@ def assert_good(username)
     end
   end
 
+  describe "update_timezone_if_missing" do
+    let(:timezone) { nil }
+
+    it "does nothing if timezone is nil" do
+      UserOption.expects(:select).never
```

Comment:
We prefer to avoid stubbing. For example I suggested an update above instead of a select, which would allow this test to pass even if it wasn't working.

It would be better if you made a user object with at timezone, called this method and then checked that the timezone didn't change.

Reply:
No problem :+1: